nvidia-apply-extra.c

```diff
@@ -99,6 +99,7 @@ static int
 should_extract (struct archive_entry *entry)
 {
   const char *path = archive_entry_pathname (entry);
+  char new_path[1024];
```

Comment:
```suggestion
  char new_path[PATH_MAX];
```

Reply:
The `1024` value was pretty random. Thanks for the hint, fixed.